config/app/widgets.yaml

```diff
@@ -82,6 +82,12 @@ parameters:
         params:
           procedure: 'non_decence_energetique'
           status: 'en_cours'
+      cardPartenairesNonNotifiables:
+        label: "Partenaires non notifiables"
+        roles: [ROLE_ADMIN, ROLE_ADMIN_TERRITORY]
+        link: 'back_partner_index'
+        params:
+          isNotNotifiable: 1
```

Comment:
bon le nom de la variable m'embête (avec un Not) mais j'ai pas mieux à proposer ; je bloquerai pas pour ça :D

Reply:
mouais j'ai pas trouvé mieux non plus